hyrax.gemspec

```diff
@@ -51,19 +51,19 @@ EOF
   spec.add_dependency 'legato', '~> 0.3'
   spec.add_dependency 'posix-spawn'
   spec.add_dependency 'jquery-ui-rails', '~> 5.0'
-  spec.add_dependency 'redis-namespace', '~> 1.5.2'
-  spec.add_dependency 'flot-rails', '~> 0.0.6'
+  spec.add_dependency 'redis-namespace', '~> 1.5'
+  spec.add_dependency 'flot-rails', '~> 0.0'
   spec.add_dependency 'almond-rails', '~> 0.1'
   spec.add_dependency 'qa', '~> 1.0' # questioning_authority
   spec.add_dependency 'flipflop', '~> 2.3'
-  spec.add_dependency 'jquery-datatables-rails', '~> 3.4.0'
+  spec.add_dependency 'jquery-datatables-rails', '~> 3.4'
   spec.add_dependency 'rdf-rdfxml' # controlled vocabulary importer
   spec.add_dependency 'clipboard-rails', '~> 1.5'
   spec.add_dependency 'rails_autolink', '~> 1.1'
   spec.add_dependency 'active_fedora-noid', '~> 2.0', '>= 2.0.2'
   spec.add_dependency 'awesome_nested_set', '~> 3.1'
   spec.add_dependency 'breadcrumbs_on_rails', '~> 3.0'
-  spec.add_dependency 'kaminari_route_prefix', '~> 0.1.1'
+  spec.add_dependency 'kaminari_route_prefix', '~> 0.1'
```

Comment:
same here. Please keep the pin as it was before, lest I release a breaking change 😆

Reply:
Will do. Keeping my eye on you.